includes/widgets/common.php

```diff
@@ -43,6 +43,51 @@ public function show_in_panel() {
 		return false;
 	}
 
+	/**
+	 * @param $shape String Shape name.
+	 *
+	 * @return string The shape path in the assets folder.
+	 */
+	private function get_shape_url( $shape ) {
+		return ELEMENTOR_ASSETS_URL . 'mask-shapes/' . $shape . '.svg';
```

Comment:
Maybe use one JSON file for all shapes? like here https://github.com/elementor/elementor/pull/13876/files#diff-9f05d4843d392f8d215c6a90fb7717b02f363928fca1cb60419fddaa5145fe6dR251-R252

Reply:
It has to return a URL in order to be able to pass it to the `mask-image`. 
If we were injecting the SVG markup into the page that would have been a good solution.